src/Runner.Common/JobServerQueue.cs

```diff
@@ -230,21 +232,37 @@ public void QueueFileUpload(Guid timelineId, Guid timelineRecordId, string type,
             _fileUploadQueue.Enqueue(newFile);
         }
 
-        public void QueueSummaryUpload(Guid stepRecordId, string name, string path, bool deleteSource)
+        public void QueueResultsUpload(Guid timelineRecordId, string name, string path, string type, bool deleteSource, bool finalize, bool firstBlock, long totalLines)
         {
+            if (!_resultsClientInitiated)
+            {
+                Trace.Verbose("Skipping results upload");
```

Comment:
should we also delete the file base on `deleteSource==true` when we skip?

Reply:
Good call, deleted file here.